src/runtime/virtcontainers/device/config/config.go

```diff
@@ -263,9 +263,15 @@ type VFIODev struct {
 	// Guest PCI path of device
 	GuestPciPath vcTypes.PciPath
 
+	// APDevices are the Adjunct Processor devices in the mdev. Mutually exclusive with PCI.
+	APDevices []string
```

Comment:
Why multiple devices here?  For the PCI, `VFIODev` represents a single PCI device, with multiple devices in the VFIO group being represented by multiple `VFIODev`s in a single `VFIODevice` (yes, the names are confusing).  Shouldn't the same model apply here?

Or, better yet `VFIODevice` could list either a bunch of `PCIDev`s or a bunch of `APDev`s, improving the naming and reducing redundancy in the `VFIODev` structure.